content/sensu-go/5.2/reference/agent.md

```diff
@@ -138,57 +146,120 @@ ok
 description     | Returns `ok` if the agent is active and connected to a Sensu backend; returns `sensu backend unavailable` if the agent is unable to connect to a backend.
 example url     | http://hostname:3031/healthz
 
-### `/events` (POST)
+## Creating monitoring events using the agent TCP and UDP sockets
```

Comment:
I wonder if it would be useful to include the word "legacy" somewhere in this title? "legacy monitoring events" perhaps?